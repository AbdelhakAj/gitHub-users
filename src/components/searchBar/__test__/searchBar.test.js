import React from "react";
import renderer from "react-test-renderer";
import SearchBar from "../searchBar";

describe("<SearchBar />", () => {
  it("renders SearchBar", () => {
    const component = renderer.create(<SearchBar />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
