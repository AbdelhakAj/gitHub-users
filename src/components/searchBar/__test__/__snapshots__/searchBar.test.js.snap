// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> renders SearchBar 1`] = `
<div
  className="search-bar"
>
  <input
    className="search-bar-username"
    id="username"
    name="username"
    onChange={[Function]}
    placeholder="write a github username here"
    type="text"
    value=""
  />
  <button
    className="search-bar-btn"
    onClick={[Function]}
  >
    search
  </button>
</div>
`;
